bitnami/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.7.17
+version: 10.7.18
```

Comment:
As we are adding a new parameter to the values, we should bump it into a minor instead of a patch version.

Reply:
@dani8art, done.